drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -89,6 +89,8 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
     :PROJECT_CONFIG,
     :CLIENT_ALERT,
     :FORM_DEFINITION,
+    :SERVICE_TYPE,
+    :SERVICE_CATEGORY,
```

Comment:
Are you using Static Forms for these? I don't see JSON files added. IMO these are simple enough (just a name?) that we can just as well implement them custom on the frontend. That lets us be a bit more flexible with the UI. (Not tied to StaticForm).

Reply:
Ok good plan, I will implement the forms directly in the frontend.